api/wallet.go

```diff
@@ -371,13 +371,20 @@ func (srv *Server) walletTransactionHandler(w http.ResponseWriter, req *http.Req
 
 // walletTransactionsHandler handles API calls to /wallet/transactions.
 func (srv *Server) walletTransactionsHandler(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	// If 'startheight' or 'endheight' query parameters don't exist,
+	// return a descriptive error message along with a HTTP 400 Bad Request response code.
```

Comment:
generally we leave out this sort of comment, since it follows pretty clearly from the code